cmake/BuildTests.cmake

```diff
@@ -30,6 +30,16 @@ ENDMACRO()
 
 set(NANOFI_TEST_DIR "${CMAKE_SOURCE_DIR}/nanofi/tests/")
 
+function(configureTestResources SOURCE_DIR DEST_DIR)
```

Comment:
As it doesn't do any configuration maybe it should be named something like `copyTestResources`